mobile/src/api/dish/index.ts

```diff
@@ -0,0 +1,10 @@
+import {httpClient} from '../client';
+import {API_ROUTES} from '../const';
+
+export const addDish = async (dish: FormData) => {
+  const response = await httpClient.post(API_ROUTES.v1.addDish, dish, {
+    headers: {'Content-Type': 'multipart/form-data'},
+    transformRequest: () => dish,
```

Comment:
za co odpowiada ta linijka?
I planujemy tworzyć osobną warstwę do zapytań API?

Reply:
Zastanawiam się właśnie na ile to powinno być osobne zapytanie a na ile nie powinno być już spięte tutaj z hookiem useMutation? Co myślisz @victsob98

Reply:
prawdopodobnie jest to bug związany z libką, wystepuje problem przy wysyłaniu form data, więc skorzystałam z tego rozwiązania 
https://github.com/axios/axios/issues/860 , stąd dodanie tego transformRequest

Reply:
a co do drugiego komentarza, nadal wydaje mi się, że lepiej to trzymać osobno, w sensie wydaje mi się to bardziej przejrzyste, jeżeli będziemy chcieli dodać kolejne zapytanie to dodamy je w jednym pliku przeznaczonym dla dish. Nie wiem co o tym myślisz, ale ja bym jednak została przy zostawieniu tego tak jak jest teraz